plugins/resources/src/compound.cpp

```diff
@@ -533,10 +564,16 @@ namespace
 
     int close_replica(
         irods::plugin_context& _ctx,
-        const int l1descInx) {
+        const int l1descInx,
+        const int operation_status = 0) {
+
```

Comment:
Remove empty line.

Clang-Format is going to put the curly brace on its own line which will keep things clear.